tests/serializers/test_natural.py

```diff
@@ -69,6 +69,27 @@ def natural_key_test(format, self):
     self.assertIsNone(books[1].object.pk)
 
 
+def natural_primary_keys_multi_table_test(format, self):
+    sub_class_instance_one = {'code': '1'}
+    sub_class_instance_two = {'code': '2'}
+
+    SubClass.objects.create(**sub_class_instance_one)
```

Comment:
I was following standards for some tests in that same file, so I thought it was the expected standard. But I rewrote it in the more "common" way. :)